cmd/charm/charmcmd/cmd.go

```diff
@@ -86,6 +90,12 @@ type csClient struct {
 	filler esform.Filler
 }
 
+// SaveJAR calls save on the jar member variable. This follows the Law
+// of Demeter and allows csClient to meet interfaces.
+func (c *csClient) SaveJAR() error {
```

Comment:
We *really* need to pull out the common code between this and core.  (but not here)